### ISSUE TITLE
Add some documentation about how the tag-conditions for if-then theme properties work

### ISSUE THREAD

**Issue by tordans** [pietervdvn/MapComplete#887]
I found this code, that describes the possible conditions. But they are hard to parse. Can someone help to update those docs with some human readable form and maybe put it somewhere easier to find?

From `Logic/Tags/TagUtils.ts`
```
     * Parses a tag configuration (a json) into a TagsFilter
     *
     * TagUtils.Tag("key=value") // => new Tag("key", "value")
     * TagUtils.Tag("key=") // => new Tag("key", "")  // <-- this is "true if <key> is null"
     * TagUtils.Tag("key!=") // => new RegexTag("key", /^..*$/)
     * TagUtils.Tag("key~*") // => new RegexTag("key", /^..*$/) // <-- this is "true key is not null regardless of it's value"
     * TagUtils.Tag("key!=value") // => new RegexTag("key", "value", true)
     * TagUtils.Tag("vending~.*bicycle_tube.*") // => new RegexTag("vending", /^.*bicycle_tube.*$/)
     * TagUtils.Tag("x!~y") // => new RegexTag("x", /^y$/, true)
     * TagUtils.Tag({"and": ["key=value", "x=y"]}) // => new And([new Tag("key","value"), new Tag("x","y")])
     * TagUtils.Tag("name~[sS]peelbos.*") // => new RegexTag("name", /^[sS]peelbos.*$/)
     * TagUtils.Tag("survey:date:={_date:now}") // => new SubstitutingTag("survey:date", "{_date:now}")
     * TagUtils.Tag("xyz!~\\[\\]") // => new RegexTag("xyz", /^\[\]$/, true)
     * TagUtils.Tag("tags~(.*;)?amenity=public_bookcase(;.*)?") // => new RegexTag("tags", /^(.*;)?amenity=public_bookcase(;.*)?$/)
     * TagUtils.Tag("service:bicycle:.*~~*") // => new RegexTag(/^service:bicycle:.*$/, /^..*$/)
     *
     * TagUtils.Tag("xyz<5").matchesProperties({xyz: 4}) // => true // <-- this also works for date comparison, eg "check_date<2022-01-01"
     * TagUtils.Tag("xyz<5").matchesProperties({xyz: 5}) // => false
```

**Comment by pietervdvn** [comment]
(Note that in some cases, some filters are not allowed, e.g. using a regex in a preset. These errors should be automatically detected and will throw an error)

**Comment by tordans** [comment]
Thanks, will have a look.
It would be great to link this docs from the Schema. That would have helped me find it quicker.
